as/thrift.js

```diff
@@ -45,6 +45,7 @@ function TChannelAsThrift(opts) {
         'must pass source as an argument');
 
     self.thriftSource = opts.source;
+    self.thriftFileName = opts.thriftFileName || 'service.thrift';
```

Comment:
nope.

This argument must either be undefined or not.

We should only implement `thriftIDL` if this is specified.